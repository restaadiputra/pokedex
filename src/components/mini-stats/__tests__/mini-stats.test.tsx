import React from 'react';
import { render, screen } from '@testing-library/react';
import MiniStats from '../mini-stats';

test('should render the area with capitalize on first letter', () => {
  const { container } = render(<MiniStats height="100m" weight="100kg" />);
  expect(screen.getByText('Kanto')).toBeInTheDocument();
  expect(screen.getByText('100m')).toBeInTheDocument();
  expect(screen.getByText('100kg')).toBeInTheDocument();

  expect(container).toMatchSnapshot();
});
