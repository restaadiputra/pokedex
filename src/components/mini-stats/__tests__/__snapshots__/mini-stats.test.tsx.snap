// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the area with capitalize on first letter 1`] = `
<div>
  <div
    class="mb-4 bg-gray-700 relative"
  >
    <div
      class="absolute top-0 left-0 transform rotate-90 origin-bottom-left -translate-y-4 -translate-x-1"
    >
      <p
        class="text-white tracking-wide text-sm opacity-75 font-light"
      >
        <span
          class="font-normal"
        >
          Region : 
        </span>
        Kanto
      </p>
    </div>
    <div
      class="ml-16 mt-12"
    >
      <p
        class="text-white tracking-wide text-sm opacity-75 font-light"
      >
        <span
          class="font-normal"
        >
          Height : 
        </span>
        100m
      </p>
      <p
        class="text-white tracking-wide text-sm opacity-75 font-light"
      >
        <span
          class="font-normal"
        >
          Weight : 
        </span>
        100kg
      </p>
    </div>
  </div>
</div>
`;
